medicines/doc-index-updater/src/models.rs

```diff
@@ -210,21 +214,32 @@ impl FromStr for DeleteMessage {
     }
 }
 
+#[async_trait]
 impl Message for CreateMessage {
+    fn get_id(&self) -> Uuid {
+        self.job_id
+    }
```

Comment:
Can this (and `to_json_string`) be provided as a default implementation for the `Message` trait? I think I want to say "derivable".

Reply:
I've looked at both, but I couldn't make `to_json_string` universal because I can't make `Message` trait require the `Deserialize` trait without having to play around with lifetimes, and I couldn't see any way to make a trait demand a property